setup.py

```diff
@@ -94,10 +94,12 @@ def err(*args, **kwargs):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Software Development',
         'Topic :: Scientific/Engineering',
         'Topic :: Utilities',
     ],
+    install_requires=install_requires,
```

Comment:
easy_install is [mentioned](https://github.com/titusjan/argos/blob/254963c3a466490e8f595692125707c2e51e1aa9/setup.py#L50-L51), though.